node/src/actors/chain_manager/mod.rs

```diff
@@ -703,6 +708,17 @@ impl ChainManager {
                         JsonRpcServer::from_registry().do_send(BlockNotify { block })
                     }
                 }
+
+                if miner_pkh == own_pkh {
+                    self.chain_state.node_stats.block_mined_count += 1;
+                    if self.sm_state == StateMachine::Synced {
+                        log::info!("Congratulations! Your block was consolidated into the block chain by an apparent majority of peers");
+                    } else {
+                        // During synchronization, we assume that every consolidated block has,
+                        // at least, one proposed block.
+                        self.chain_state.node_stats.block_proposed_count += 1;
```

Comment:
I thought this is the count of blocks proposed by this node? So it should be assumed to be 0 unless the miner pkh is the same as the node pkh?

Reply:
block_proposed_count is a counter for all the proposed blocks by the node, in order that you can check how many times you create and broadcast a valid block and how many times it was included it in the blockchain, the block_mined_count

Reply:
In this case, if you detect a mined block by you during synchronization, you assume that you proposed a candidate in the past

Reply:
Ok, so I would assume that the number of proposed blocks is 0, because the probability of proposing a block is very low. Right?

Reply:
Ah, nevermind, I missed the `if miner_pkh == own_pkh`